lib/KImg.vue

```diff
@@ -81,6 +82,15 @@
         default: undefined,
       },
     },
+    emits: {
```

Comment:
This would be great, but it seems it's only available for Vue 3 https://v3-migration.vuejs.org/breaking-changes/emits-option.html? Doesn't seem to be breaking anything but I think it'd be best to remove it.

When you place the documentation string directly above the emit like this
```
/**
 * Emitted when the image fails to load
 * /
this.$emit('error', event);
```

I think our documentation generator will pick it up. It may also be nice to document that `event` is available in the payload (just in case consumers needed it)

Reply:
Got it! Will make the necessary changes!